components/02-molecules/menu/_menu-item.twig

```diff
@@ -14,29 +19,44 @@
   {% set item__modifiers = item__modifiers|merge(['with-sub']) %}
 {% endif %}
 
-{% import "@molecules/menu/menu.twig" as menus %}
+{# @TODO: When we build this in Drupal. Verify this works #}
+{# @LINK: https://yaleits.atlassian.net/browse/YALB-352 #}
+{% if item.is_active %}
+  {% if breadcrumb %}
+    {% set item__attributes = item__attributes|merge({
+      'aria-current': 'page',
+    }) %}
+  {% else %}
+    {% set link__attributes = link__attributes|merge({
+      'aria-current': 'page',
+    }) %}
+  {% endif %}
+{% endif %}
 
 {# Set link component as default #}
 {% set list__item %}
-  {% block list__item__content_block %}
+  {% if breadcrumb and item.is_active %}
```

Comment:
Checking for `breadcrumb` here and above within this file seems less than ideal. For instance, you'll probably need to do the same thing for the active items for pagers.

I think you could probably leave the if statement but just check for `item.in_active_trail`. Then ditch the `item__attributes` array altogether and pass this to the link component below and wrap line 39 with a span `<span add_attributes(link__attributes)>{{- item.title -}}</span>`

Reply:
Good ideas! I've updated the code per our zoom conversation.